Resources/doc/index.rst

```diff
@@ -97,6 +97,7 @@ application:
 
 - The ``services`` node allows you to provide custom services to the underlying ``DependencyFactory`` part of ``doctrine/migrations``.
 - The node ``factories`` is similar to ``services``, with the difference that it accepts only callables.
+
```

Comment:
@goetas When merging up this blank line is important (it's not missing in the lower branch).